guides/changing-metadata-store.md

```diff
@@ -0,0 +1,14 @@
+# Changing Metadata Store
+
+Aqueduct uses a storage layer for tracking workflow, operator, and artifact metadata. The default storage layer is the local filesystem. Aqueduct currently also supports using the following services for its metadata store:
+- AWS S3
+- Google Cloud Storage (GCS)
+
+The Aqueduct UI should be used to change the metadata store to AWS S3 or GCS. You
+do this by connecting a new integration of the respective service. As part of the
+integration connection dialog, you must select the box at the bottom that says
+`Use this integration for Aqueduct metadata storage.`.
+
+![](<../.gitbook/assets/change_metadata_store.png>)
```

Comment:
For the image, can we take a screenshot of the full connection UI and then grey out everything but the box?

Reply:
Is there a specific way you want me to do this? Just grey out the text?